style.css

```diff
@@ -424,6 +529,84 @@ body {
 }
 
 @media screen and (min-width: 768px) {
+  .detail {
+    display: none;
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 70%;
+    height: 600px;
+    padding: 20px 30px;
+    box-shadow: 0 1px 10px #344;
+    background: #fff;
+    border-radius: 10px;
+    overflow-y: auto;
+    font-family: 'Poppins', sans-serif;
+    z-index: 20;
+  }
```

Comment:
- [ ] Your popup window is working properly but have some issue to implement the following : 

- When I click the see project button it is appear but the background should be blur not only desktop version and but also mobile version to align `Figma` design and look good and check the below the picture.

**Your Design**

![Screenshot (202)](https://github.com/mioOleen/Portfolio-1/assets/118582316/b88ce7ee-253b-4a28-97a7-1e1879bcd817)

**Figma Design**

![Screenshot (203)](https://github.com/mioOleen/Portfolio-1/assets/118582316/1279a170-ab18-4bf1-8a3c-02613377f4e0)